### ISSUE TITLE
Move listbox checkbox to classes

### ISSUE THREAD

**Issue by j4587698** [kikipoulet/SukiUI#16]
Now listbox has a default checkbox, but sometimes don't need checkbox.
Maybe can move checkbox to a classes?

**Comment by kikipoulet** [comment]
The default ListBoxItem has been changed on 4.5.4 Nuget version, without checkbox.

Can you show me your listbox xaml so I can quickly test it ?

**Comment by j4587698** [comment]
Sorry, I used 4.5.3, I'll try 4.5.4.tks

**Comment by kikipoulet** [comment]
No problem, the suggestion to remove the checkbox was pertinent I appreciate :+1: 

On 4.5.4 it should have this style :

![screen](https://user-images.githubusercontent.com/19242427/211169897-9a6e2179-83a4-475d-8dd4-e3a46691f76d.jpg)

**Comment by j4587698** [comment]
Now ListboxItem VerticalAlignment is center
```
    <Style Selector="ListBoxItem">
        <Setter Property="Background" Value="Transparent" />
        <Setter Property="Padding" Value="4,2" />
        <Setter Property="Template">
            <ControlTemplate>
                <Border
                    CornerRadius="6"
                    Margin="0,0,10,0"
                    Name="BorderBasicStyle"
                    Padding="8,4">
                    <DockPanel>
                        <avalonia:MaterialIcon
                            DockPanel.Dock="Right"
                            Foreground="{DynamicResource SukiPrimaryColor}"
                            Kind="Check"
                            Name="CheckSelected" />
                        <ContentPresenter
                            Background="Transparent"
                            BorderBrush="Transparent"
                            BorderThickness="{TemplateBinding BorderThickness}"
                            Content="{TemplateBinding Content}"
                            ContentTemplate="{TemplateBinding ContentTemplate}"
                            CornerRadius="{TemplateBinding CornerRadius}"
                            HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}"
                            Margin="0,0,0,0"
                            Name="PART_ContentPresentern"
                            Padding="{TemplateBinding Padding}"
                            VerticalAlignment="Center"
                            VerticalContentAlignment="Center" />
                    </DockPanel>
                </Border>
            </ControlTemplate>
        </Setter>
```

Can change to properties like HorizontalContentAlignment ?

In WrapPanel, VerticalAlignment = "Top" maybe more common.

**Comment by kikipoulet** [comment]
I don't really understand what behavior is problematic here, can you show me with an image how it looks now and how you think it should be ?

**Comment by j4587698** [comment]
I have a ListBox like 
```
    <ListBox Name="BookList" Items="{Binding Books}" SelectionChanged="BookList_OnSelectionChanged">
        <ListBox.ItemsPanel>
                <ItemsPanelTemplate>
                    <WrapPanel></WrapPanel>
                </ItemsPanelTemplate>
            </ListBox.ItemsPanel>
            <ListBox.ItemTemplate>
                <DataTemplate>
                    <StackPanel Orientation="Vertical" Width="110" VerticalAlignment="Top">
                        <Image Width="110" Source="{Binding ImageBase64, Converter={StaticResource ImageConverter}}"></Image>
                        <TextBlock Text="{Binding BookName}" HorizontalAlignment="Center" TextWrapping="Wrap"></TextBlock>
                    </StackPanel>
                </DataTemplate>
            </ListBox.ItemTemplate>
        </ListBox>
```

The Listbox has a Image and a title textblock.

![image](https://user-images.githubusercontent.com/24642446/211192766-c5c64876-9a41-49ef-9c83-4b336fded956.png)

The Textblock maybe have more than one line.
The Listbox will align center.
But in this demo, I want to top align.
I try add `VerticalAlignment="Top"` to `StackPanel ` but it not work.

**Comment by kikipoulet** [comment]
Oh I get it, because of that your images can't be aligned. 

I'll try to solve it in the Nuget package of tomorrow

**Comment by kikipoulet** [comment]
Solved in nuget package 4.5.5

Note : Selected item background is now gray for style purpose
